products/template/tests/test_build.py

```diff
@@ -6,22 +6,6 @@
 from build_scripts import PRODUCT_S3_NAME, BUILD_NAME, PG_CLIENT
 
 
-# * test Load stage
-@pytest.mark.end_to_end()
-def test_load_db_tables():
-    expected_source_tables = [
-        "nypl_libraries",
-        "bpl_libraries",
-        "qpl_libraries",
-        "dpr_parksproperties",
-        "dpr_greenthumb",
-        "lpc_landmarks",
-    ]
-    actual_tables = PG_CLIENT.get_schema_tables()
-    for table in expected_source_tables:
-        assert table in actual_tables
-
-
 # * test Transform stage
 @pytest.mark.end_to_end()
 def test_transform_staging():
```

Comment:
Would you want dbt to cover this as well?

Reply:
true, I just chose to start with the output-end and save these for later. the [`_sources.yml`](https://github.com/NYCPlanning/data-engineering/blob/dm-e-to-e-product-test/products/template/sql/_sources.yml#L11) is also incomplete

happy to cover all tables in this PR if it seems worth it

Reply:
Nah totally up to you

Reply:
I'll hold off on covering all staging and source tables